### DISCUSSION TITLE
QR Controll App - End Time does not work

### DISCUSSION THREAD

**Discussion by Kim-Ocean** [gopro/labs#363] [Q&A] [answered]
Dear Forum members,

I am using the QR Control app on an iPhone together with a GoPro Hero 11.
I would like to create a QR code for time lapse photo in linear mode, raw=off, taking a picture every two seconds - that so far works

Now I would like to enter a start and end time:
The entered start time works perfectly fine but the camera does not take my end time (I tired with several different end times from a few seconds to minutes to hours). The camera uses the end time, which is already set in the camera configuration but not the one I chose for the QR code. Can anyone tell me how I can solve this problem so that the camera also reads the end time from the QR code?

Thank you for your help!!

**Comment by dnewman-gpsw** [comment, marked as answer]
Share the command you are trying.

**Reply by Kim-Ocean** [reply]
The QR command I am using is: !5NmTPp.2fLr0!S!600E

I changed yesterday a bit on the time and date settings (24h and DD/MM/YYYY) and now it seems to work?!

I will now run some more trials to be sure....maybe it helps someone else or somebody nows if that could have interfered with the QR code functions.

Thanks
